### PR TITLE
[9.0][ADD] purchase_stock_picking_return_invoicing_open_qty

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Purchase Stock Picking Return Invoicing Open Qty
================================================

This module makes compatible the modules 'Purchase Open Qty' and
'Purchase Stock Picking Return Invoicing'.

* This module will be removed in version 10.0

### CODE REVIEW

File: purchase_stock_picking_return_invoicing_open_qty/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================================================
+Purchase Stock Picking Return Invoicing Open Qty
+================================================
+
+This module extends makes compatible the modules 'Purchase Open Qty' and
+'Purchase Stock Picking Return Invoicing'.
```
Comment (JordiBForgeFlow) [inline purchase_stock_picking_return_invoicing_open_qty/README.rst]: Please indicate where can the module 'purchase_open_qty' be obtained from. (OCA/purchase-workflow)

File: purchase_stock_picking_return_invoicing_open_qty/README.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/95/9.0
+
+Known issues / Roadmap
```
Comment (JordiBForgeFlow) [inline purchase_stock_picking_return_invoicing_open_qty/README.rst:19]: Indicate here that this module will be removed in 10.0.

Comment (MiquelRForgeFlow) [reply purchase_stock_picking_return_invoicing_open_qty/README.rst:19]: Done

**Comment by MiquelRForgeFlow** [comment]
@pedrobaeza @jbeficent  This glue module should be merged ASAP, because the modules https://github.com/OCA/account-invoicing/tree/9.0/purchase_stock_picking_return_invoicing and https://github.com/OCA/purchase-workflow/tree/9.0/purchase_open_qty are both fixed and already merged.

**Review by JordiBForgeFlow — APPROVED** [review]

File: purchase_stock_picking_return_invoicing_open_qty/README.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+======================
+
+* This module will be removed in version 10.0, because the module
+'Purchase Stock Picking Return Invoicing' will already be compatible.
```
Comment (pedrobaeza) [inline purchase_stock_picking_return_invoicing_open_qty/README.rst]: Incorrect indentation: it needs 2 spaces

**Comment by MiquelRForgeFlow** [comment]
@pedrobaeza ready!

**Pull request merged**